libcontainer/selinux/selinux.go

```diff
@@ -158,7 +158,9 @@ func Setfilecon(path string, scon string) error {
 // Getfilecon returns the SELinux label for this path or returns an error.
 func Getfilecon(path string) (string, error) {
 	con, err := system.Lgetxattr(path, xattrNameSelinux)
-
+	if err != nil {
+		return "", err
+	}
```

Comment:
Then probably change the final return to `return string(con), nil` in this function?